cmake/FindTBB.cmake

```diff
@@ -96,6 +96,14 @@ if(NOT TBB_FOUND)
     endif()
   endif()
   
+  ##################################
+  # Try TBBConfig.cmake first
+  ##################################
+  find_package(TBB CONFIG)
+  if(TBB_FOUND)
+    return()
```

Comment:
we need something like this before returning
```cmake
set(TBB_LIBRARIES TBB::tbb)
```
Otherwise in CONFIG mode `TBB_LIBRARIES` is empty and it won't link with tbb